include/obj_loader.h

```diff
@@ -0,0 +1,5 @@
+#include "types.h"
+
+using MeshList = std::vector<Mesh>;
+using MeshInstances = std::vector<MeshInstance>;
```

Comment:
To avoid weird type-related confusion can you move these to `types.h` and also remove them from `gltf_loader.h`?
```suggestion
```